.travis.yml

```diff
@@ -14,11 +18,11 @@ jobs:
   include:
   - stage: test
     script:
-    - cd $HOME/gopath/src/github.com/integr8ly/keycloak-operator && make setup test
+    - cd $HOME/gopath/src/github.com/integr8ly/$OPERATOR_NAME && make setup test
   - stage: deploy
     script:
     - docker login --password "$QUAY_PASSWORD" --username "$QUAY_USERNAME" quay.io
-    - cd $HOME/gopath/src/github.com/integr8ly/keycloak-operator
-    - make compile COMPILE_TARGET=./tmp/_output/bin/keycloak-operator
-    - docker build -t quay.io/integreatly/keycloak-operator:$(git rev-parse --short ${TRAVIS_COMMIT}) -f tmp/build/Dockerfile .
-    - docker push quay.io/integreatly/keycloak-operator:$(git rev-parse --short ${TRAVIS_COMMIT})
+    - cd $HOME/gopath/src/github.com/integr8ly/$OPERATOR_NAME
+    - make compile COMPILE_TARGET=./tmp/_output/bin/$OPERATOR_NAME
+    - docker build -t quay.io/integreatly/$OPERATOR_NAME:$(git rev-parse --short ${TRAVIS_COMMIT}) -f tmp/build/Dockerfile .
+    - docker push quay.io/integreatly/$OPERATOR_NAME:$(git rev-parse --short ${TRAVIS_COMMIT})
```

Comment:
Would it make more sense to add a build-and-push target to all Makefiles if it doesn't already exist and use that in all travis files making them all the same?

```
docker login --password "$QUAY_PASSWORD" --username "$QUAY_USERNAME" quay.io
cd $HOME/gopath/src/github.com/integr8ly/$OPERATOR_NAME
TAG=$(git rev-parse --short ${TRAVIS_COMMIT}) make build-and-push
```

This way we leave the logic of how it gets compiled/built in the makefile which can actually be different for some of them see https://github.com/integr8ly/3scale-operator/blob/master/Makefile#L36, and also allows us to easily do it manually if the need should arise.

Reply:
yep, this sounds good to me, I'll do it now

Reply:
Probably going to be issues here if we cant install things in the environment. The likes of 3scale expects an additional tool to be installed to do the packr stuff. Can we install a go package easily enough?